tests/Unit/DependencyInjection/FOSHttpCacheExtensionTest.php

```diff
@@ -662,10 +665,62 @@ public function testVarnishCustomTagsHeader()
         $this->assertEquals(['tag_mode' => 'ban', 'tags_header' => 'myheader'], $container->getParameter('fos_http_cache.proxy_client.varnish.options'));
     }
 
+    /**
+     * @param string|array|mixed $serversValue           string and array are ok, anything else not, string only if it uses an env var
+     * @param string|mixed|null  $envValue               should be a string, otherwise an error will show up
+     * @param array|null         $expectedServersValue
+     * @param string|null        $expectExceptionClass
+     * @param string|null        $expectExceptionMessage
+     *
+     * @dataProvider dataVarnishServersConfig
+     */
+    public function testVarnishServersConfig($serversValue, $serversFromJsonEnvValue, $envValue, $expectedServersValue, $expectExceptionClass, $expectExceptionMessage)
+    {
+        if ($expectExceptionClass) {
```

Comment:
the exceptions should be thrown on the line with `$this->extension->load([$config], $container);`, right? lets move it to right before that line to not get confusing reports if an exception is thrown before then.

Reply:
you mean, in case the test preparation is throwing an exception? ok.